.github/workflows/ci.yml

```
@@ -0,0 +1,206 @@
+
+# Allow only one concurrent deployment per branch
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  # Job 1: Static Analysis & Code Metrics
+  analyze:
+    name: 📊 Static Analysis & Code Metrics
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)